deploy/prod/ampc-hnsw-0-prod/values-ampc-hnsw.yaml

```diff
@@ -0,0 +1,193 @@
+env:
+  - name: RUST_LOG
+    value: "info"
+
+  - name: RUST_BACKTRACE
+    value: "full"
+
+  - name: SMPC__ENVIRONMENT
+    value: "prod"
+
+  - name: SMPC__SERVICE__SERVICE_NAME
+    value: "hnsw-service-0"
+
+  - name: SMPC__PARTY_ID
+    value: "0"
+
+  - name: SMPC__CPU_DATABASE__URL
+    valueFrom:
+      secretKeyRef:
+        key: DATABASE_AURORA_HNSW_URL
+        name: application
+
+  - name: SMPC__MAX_DB_SIZE
+    value: "2000000"
+
+  - name: SMPC__MAX_BATCH_SIZE
+    value: "32"
+
+  - name: SMPC__PROCESSING_TIMEOUT_SECS
+    value: "240"  # 2 minutes per batch in stage, bump to 4 in prod
+
+  - name: SMPC__HAWK_REQUEST_PARALLELISM
+    value: "1024"
+
+  - name: SMPC__HAWK_CONNECTION_PARALLELISM
+    value: "8"
+
+  - name: SMPC__HAWK_STREAM_PARALLELISM
+    value: "32"
+
+  - name: SMPC__DISABLE_PERSISTENCE
+    value: "false"
+
+  - name: SMPC__MATCH_DISTANCES_BUFFER_SIZE
+    value: "512"
+
+  - name: SMPC__N_BUCKETS
+    value: "16"
+
+  - name: SMPC__ENABLE_SENDING_ANONYMIZED_STATS_MESSAGE
+    value: "false"
+
+  - name: SMPC__HAWK_SERVER_REAUTHS_ENABLED
+    value: "true"
+
+  - name: SMPC__HAWK_SERVER_RESETS_ENABLED
+    value: "false"
+
+  - name: SMPC__HAWK_SERVER_DELETIONS_ENABLED
+    value: "false"
+
+  - name: SMPC__LUC_ENABLED
+    value: "true"
+
+  - name: SMPC__LUC_LOOKBACK_RECORDS
+    value: "5"
+
+  - name: SMPC__LUC_SERIAL_IDS_FROM_SMPC_REQUEST
+    value: "false"
+
+  - name: SMPC__AWS__REGION
+    value: "eu-central-1"
+
+  - name: SMPC__SERVICE_PORTS
+    value: '["4000","4001","4002"]'
+
+  - name: SMPC__HAWK_SERVER_HEALTHCHECK_PORT
+    value: '3000'
+
+  - name: SMPC__NODE_HOSTNAMES
+    value: '["0.0.0.0","node.1.prod.hnsw.worldcoin.org","node.2.prod.hnsw.worldcoin.org"]'
+
+  - name: SMPC__SHARES_BUCKET_NAME
```

Comment:
No bucket of that name is present. I can see only `wf-mpc-prod-smpcv2-sns-requests` bucket in vpc-prod account